### PR TITLE
[562] Views count

### PR DESCRIPTION
**Pull request by mmiedlarz**
trello card:
https://trello.com/c/JHHQqFy3/562-timebox-2h-user-should-see-increasing-viewcount-while-navigating-between-creations

### CODE REVIEW

File: CreatubblesAPIClient/Sources/Requests/Creation/CreationViewsCountIncrementResponseHandler.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+
+class CreationViewsCountIncrementResponseHandler: ErrorResponseHandler {
```
Comment (sebastianewak) [inline CreatubblesAPIClient/Sources/Requests/Creation/CreationViewsCountIncrementResponseHandler.swift:26]: Couldn't we just use `ErrorResponseHandler` if this implementation is empty?

File: CreatubblesAPIClient/Sources/Requests/Gallery/GalleryViewsCountIncrementResponseHandler.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+
+class GalleryViewsCountIncrementResponseHandler: ErrorResponseHandler {
```
Comment (sebastianewak) [inline CreatubblesAPIClient/Sources/Requests/Gallery/GalleryViewsCountIncrementResponseHandler.swift:26]: Can't we just use ErrorResponseHandler if this implementation is empty?

File: CreatubblesAPIClient/Sources/Requests/Creation/CreationViewsCountIncrementResponseHandler.swift (hunk 1 above)
Comment (mmiedlarz) [reply CreatubblesAPIClient/Sources/Requests/Creation/CreationViewsCountIncrementResponseHandler.swift:26]: We're following this approach from the very beginning. It's the convention which is helpful with writing integration tests

**Review by sebastianewak — APPROVED** [review]
Code: +1
Func: n\a

**Review by JakubJankowski — APPROVED** [review]
👍

**Pull request merged**